.github/workflows/integrate.yaml

```diff
@@ -109,6 +109,10 @@ jobs:
           - "7.3"
           - "7.4"
           - "8.0"
+        symfony-version:
+          - "4.4.*"
+          - "5.3.*"
+          - "6.0.*"
```

Comment:
Hi, thank you for your PR.
I've pushed to your branch the removal of `vimeo/psalm` from tests build so circular reference gets out of the way.
It seems you are experienced with GA workflows: could you try to write a matrix where Symfony 6 is installed only on PHP versions it supports please?